pyproject.toml

```diff
@@ -1,3 +1,20 @@
 [build-system]
 requires = ["setuptools", "versioneer==0.29"]
 build-backend = "setuptools.build_meta"
+
+[tool.ruff]
+target-version = "py38"
+
+[tool.ruff.lint]
+select = [
+    "E",  # pycodestyle errors
+    "W",  # pycodestyle warnings
+    "F",  # pyflakes
+    "I",  # isort
+    "B",  # flake8-bugbear
```

Comment:
@frichtarik maybe just comment out these stricter linters for this PR. It is what's causing CI to fail https://github.com/andhus/scantree/actions/runs/8719008751/job/23917722126?pr=21#step:4:123 I'll fix those opinionated extra checks in a follow-up PR :)

Reply:
_Or_ add a `# noqa` after the one failing line:
```
tests/test_node.py:87:13: B018 Found useless expression. Either assign it to a variable or remove it.
Found 1 error.
```

Reply:
done